src/Cli/dotnet/commands/dotnet-workload/search/WorkloadSearchVersionsCommand.cs

```diff
@@ -100,6 +90,30 @@ public override int Execute()
                     Reporter.WriteLine(string.Join('\n', versions));
                 }
             }
+            else if (_workloadVersion.Contains('@'))
```

Comment:
Because of this usage, I don't think _workloadVersion is really a good name anymore. I'm leaving it for now for two reasons:
1. We might switch to --component (or something) later, in which case changing it is unnecessary.
2. Having two possible meanings means I'm not quickly coming up with a good name.

We should change it (if we don't switch to --component) before merging.

Reply:
Yeah, it seems we are kind of mixing up different commands.  It looks like currently `dotnet workload search version` gives you a list of available workload set versions for the current feature band, `dotnet workload search version 9.0.100` gives you the manifest versions for the specified workload set version (same as the rollback file basically), and `dotnet workload search version maui@9.0.0` gives you the workload set versions with the specified manifest version that contains the specified workload.

@baronfel What do you think?  Should we be designing these commands differently?